### ISSUE TITLE
V1.1.5: exception when viewing queue

### ISSUE THREAD

**Issue by erkr** [music-assistant/hass-music-assistant#186]
I think that viewing the queue r triggers this warning and exception: 

```
2022-05-18 13:01:53 WARNING (MainThread) [music_assistant.mass.metadata] Unable to get musicbrainz ID for artist DC !
2022-05-18 13:01:58 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/config/custom_components/mass/entity.py", line 72, in __on_mass_update
    await self.async_on_update()
  File "/config/custom_components/mass/media_player.py", line 235, in async_on_update
    media_album_artist = media_item.album.artist.name
AttributeError: 'NoneType' object has no attribute 'name'
2022-05-18 13:01:58 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/config/custom_components/mass/entity.py", line 72, in __on_mass_update
    await self.async_on_update()
  File "/config/custom_components/mass/media_player.py", line 235, in async_on_update
    media_album_artist = media_item.album.artist.name
AttributeError: 'NoneType' object has no attribute 'name'
2022-05-18 13:01:58 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/config/custom_components/mass/entity.py", line 72, in __on_mass_update
    await self.async_on_update()
  File "/config/custom_components/mass/media_player.py", line 235, in async_on_update
    media_album_artist = media_item.album.artist.name
AttributeError: 'NoneType' object has no attribute 'name'
2022-05-18 13:01:59 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/config/custom_components/mass/entity.py", line 72, in __on_mass_update
    await self.async_on_update()
  File "/config/custom_components/mass/media_player.py", line 235, in async_on_update
    media_album_artist = media_item.album.artist.name
AttributeError: 'NoneType' object has no attribute 'name'
2022-05-18 13:01:59 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/config/custom_components/mass/entity.py", line 72, in __on_mass_update
    await self.async_on_update()
  File "/config/custom_components/mass/media_player.py", line 235, in async_on_update
    media_album_artist = media_item.album.artist.name
AttributeError: 'NoneType' object has no attribute 'name'
2022-05-18 13:01:59 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/config/custom_components/mass/entity.py", line 72, in __on_mass_update
    await self.async_on_update()
  File "/config/custom_components/mass/media_player.py", line 235, in async_on_update
    media_album_artist = media_item.album.artist.name
AttributeError: 'NoneType' object has no attribute 'name'
2022-05-18 13:01:59 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/config/custom_components/mass/entity.py", line 72, in __on_mass_update
    await self.async_on_update()
  File "/config/custom_components/mass/media_player.py", line 235, in async_on_update
    media_album_artist = media_item.album.artist.name
AttributeError: 'NoneType' object has no attribute 'name
```

**Comment by marcelveldt** [comment]
should be fixed with 1.1.7

**Comment by erkr** [comment]
Seems fixed indeed, thanks 😊
